### PR TITLE
Add commandbox key listener

### PR DESCRIPTION
**Pull request by johnervan**

### CODE REVIEW

File: src/main/java/tars/ui/CommandBox.java [hunk 1]
```diff
@@ -2,20 +2,26 @@
 
 import com.google.common.eventbus.Subscribe;
 
+import javafx.beans.value.ObservableValue;
```
Comment (weikangchia) [inline src/main/java/tars/ui/CommandBox.java]: @johnervan I think this import is unused and you can remove it.

File: src/main/java/tars/ui/CommandBox.java [hunk 2]
```diff
@@ -69,12 +83,28 @@ public void handle(KeyEvent ke) {
...
+        commandTextField.textProperty().addListener((observable, oldValue, newValue) -> {
+            if (newValue.equals(RsvCommand.COMMAND_WORD)) {
+                raise(new CommandBoxTextFieldValueChangedEvent(newValue));
             }
         });
+        
     }
-
+    
+    //@@author
```
Comment (weikangchia) [inline src/main/java/tars/ui/CommandBox.java:100]: Did you forget to write your matric no?

**Review by weikangchia — CHANGES_REQUESTED** [review]

File: src/main/java/tars/ui/CommandBox.java (hunk 2 above)
Comment (johnervan) [reply src/main/java/tars/ui/CommandBox.java:100]: Nope, we are supposed to type that to indicate the end of the code that we've written

File: src/main/java/tars/ui/CommandBox.java (hunk 1 above)
Comment (johnervan) [reply src/main/java/tars/ui/CommandBox.java]: @weikangchia oh yah i've removed it. haven't pushed yet. Thanks btw!! (:

File: src/main/java/tars/ui/CommandBox.java (hunk 2 above)
Comment (jaeoheeail) [reply src/main/java/tars/ui/CommandBox.java:100]: I never knew thst

**Review by weikangchia — APPROVED** [review]

**Pull request merged**